storage/rocksdb/get_rocksdb_files.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+if [ -z "$PATH" ]; then
+    export PATH="/sbin:/usr/sbin:/bin:/usr/bin"
```

Comment:
unrelated changes?

Reply:
This branch contains a commit that fixes the build for me. I'll remove it once it is merged, until then, please review 3dae3ef